tomviz/vtkNonOrthoImagePlaneWidget.cxx

```diff
@@ -362,7 +366,7 @@ void vtkNonOrthoImagePlaneWidget::SetEnabled(int enabling)
     this->Interactor->RemoveObserver(this->EventCallbackCommand);
 
     // turn off the plane
-    this->CurrentRenderer->RemoveViewProp(this->PlaneOutlineActor);
+    //this->CurrentRenderer->RemoveViewProp(this->PlaneOutlineActor);
```

Comment:
If this is no longer needed then this line and the previous line should be removed. If you think we might need it then let's resolve that before merging.